docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.9"
 
 services:
   subquery-node:
-    image: onfinality/subql-node:v2.1.3
+    image: onfinality/subql-node:latest
```

Comment:
hmm, if we are depending on the latest of this external docker image, I suggest us to add a [Docker watchtower](https://github.com/containrrr/watchtower), or a similar docker, to monitor and auto-upgrade of this container. 

I encountered problems with our Docker containers before when a dependency docker is not upgraded in time. If this subql-node upgrades, the watchtower Docker container should auto-update it and notify us in Slack. However, I just installed it on my personal server and haven’t seen it upgrade anything.

![CleanShot 2023-06-01 at 11 05 04](https://github.com/OAK-Foundation/oak-subql/assets/2616844/27998213-ab6e-4ba1-9b2b-4c8a84cedf84)

Reply:
Yes, let’s stick with the version v2.5.1. 

The screenshot I attached is the logs of the watchtower docker container. We’d use it only when we need to keep everything up-to-date.